docfx_yaml/extension.py

```diff
@@ -1384,12 +1392,51 @@ def _find_and_add_summary_details(
     if summary_type in [CLASS]:
         name_to_use = f"[{uid}]({cgc_url}{uid})"
 
-    _ENTRY_NAME_AND_ENTRY_CONTENT_BY_SUMMARY_TYPE[summary_type][1].append({
-        "uid": uid,
+        _ENTRY_NAME_AND_ENTRY_CONTENT_BY_SUMMARY_TYPE[summary_type][1].append({
+            "uid": uid,
+            "name": name_to_use,
+            "fullName": uid,
+            "isExternal": False,
+        })
+        return
+
+    # if summary_type in [METHOD, PROPERTY]:
+    short_name = yaml_data.get("name", "")
+    name_to_use = uid
+    if not (class_name := yaml_data.get("class", "")):
+        class_name = yaml_data.get("module", "")
+    anchor_name = f"#{'_'.join(class_name.split('.'))}_{short_name}"
+    # Extract the first summary line by attempting to detect the first sentence.
+    summary = yaml_data.get("summary", "")
+    first_summary_line = min(
+        summary.split(". ")[0],
+        summary.split(".\n")[0],
+        summary.split("\n\n")[0],
+        key=len,
+    )
+    if first_summary_line and first_summary_line[-1] != ".":
+      first_summary_line += "."
+
+    summary_to_use = (
+        f"{first_summary_line}\n\n"
```

Comment:
For the case where a summary isn't found, would this create extra whitespaces? I think this would become:

```
\n\n

See more: ...
```

Reply:
For methods and functions, it definitely helps to have the newlines with or without content after the syntax.
For properties & attributes, without the extra space is slightly more preferred but not a big deal. Also c.g.c will likely strip the newlines if its just blank content, it seems.